pulseapi/entries/serializers/news.py

```diff
@@ -0,0 +1,37 @@
+from rest_framework import serializers
+from ..models import Entry
+from .base import EntryWithCreatorsBaseSerializer, EntrySerializerWithCreators
+
+
+class NewsEntrySerializer(EntrySerializerWithCreators):
+    """
+    Serializes an entry based on the fact that we
+    know that it is a news item. which means:
+
+    required:
+        - id,
+        - author
+        - title
+        - url (not required in base)
+
+    useful:
+        - description
+        - issue categories
+        - tags
+        - contributors
+        - get involved
+        - get involved url
+        - thumbnail image
+
+    extraneous
+        - help categories
```

Comment:
let's actually remove help categories from this serializer. I don't think it makes sense for news. In this case we need to override the Meta class like so (and preferably use `exclude` here instead of `fields`):
```py
class Meta(EntrySerializerWithCreators.Meta):
    exclude = EntrySerializerWithCreators.Meta.exclude + ('help_categories',)
```